e2e/utilities/repo-client/apis/nodes/nodes-api.ts

```diff
@@ -214,6 +224,24 @@ export class NodesApi extends RepoApi {
     }
   }
 
+  async createNodeLink(originalNodeId: string, destinationId: string): Promise<any> {
+    const name = (await this.getNodeById(originalNodeId)).entry.name;
+    const nodeBody = {
+      name: `Link to ${name}.url`,
+      nodeType: 'app:filelink',
+      properties: {
+        'cm:destination': originalNodeId
+      }
+    }
+
+    try {
+      await this.apiAuth();
+      return await this.nodesApi.createNode(destinationId, nodeBody);
+    } catch (error) {
+      this.handleError(`${this.constructor.name} ${this.createNode.name}`, error);
```

Comment:
createNode doesn't return `NodeEntry` ? method should return Promise<NodeEntry | null> maybe?

Reply:
nope, it returns Promise<NodeEntry| any>